src/sentry/static/sentry/app/components/dropdownAutoCompleteMenu.jsx

```diff
@@ -42,7 +43,7 @@ class DropdownAutoCompleteMenu extends React.Component {
   filterItems = (items, inputValue) =>
     items.filter(item => {
       return (
-        [item.value, item.label]
+        [item.searchKey, item.label]
```

Comment:
I would recommend:

If searchKey is defined, use that, otherwise use value + label?

Reply:
done! gonna merge right after tests pass again